main.c

```diff
@@ -199,22 +201,68 @@ void updateArm()
 void raiseCarrier()
 {
 	//move carrier motors at speed so it should difinitely be able to lift the mobile goal
-	motor[carrier] = 100;
+	//for(int coefficient = 1; coefficient <= 10; coefficient++)
+	//{
+	//	motor[carrierLeft] = 10*coefficient;
+	//	motor[carrierRight] = 10*coefficient;
+	//	sleep(100);
+	//}
+	for(int base = 1; base <= 10; base++)
+	{
+		motor[carrierLeft] = pow(base,2);
+		motor[carrierRight] = -pow(base,2);
+		sleep(186);
+	}
 	//wait 1 second, needs to be tested and probably changed but this is the test value
-	sleep(1000);
+	//sleep(1000);
 	//stop carrier motors
-	motor[carrier] = 0;
+	motor[carrierLeft] = 0;
+	motor[carrierRight] = 0;
 }
 
+//lowers the mobile goal, needs to be heavily tested
 //lowers the mobile goal, needs to be heavily tested
 void lowerCarrier()
 {
+	//for(int coefficient = 1; coefficient <= 4; coefficient++)
+	//{
+	//	motor[carrierLeft] = -10*coefficient;
+	//	motor[carrierRight] = -10*coefficient;
+	//	sleep(250);
+	//}
+	for(int base = 1; base <= 10; base++)
+	{
+		motor[carrierLeft] = -(pow(base,2));
+		motor[carrierRight] = (pow(base,2));
+		sleep(105);
+	}
 	//move carrier motors at the same descending speed the arm has, 6
-	motor[carrier] = -50;
+	//motor[carrierRight] = -50;
+	//motor[carrierLeft] = -50;
 	//wait 1 second, needs to be tested and probably changed but this is the test value
-	sleep(1000);
+	//sleep(1000);
 	//stop carrier motors
-	motor[carrier] = 0;
+	motor[carrierRight] = 0;
+	motor[carrierLeft] = 0;
+}
+
+//raises or lowers the mobile goal carrier depnding if button 8D, 8R, or neither are pressed
+	//8D lifts mobile goal
+	//8R lowers mobile goal
+void updateMobileGoal()
+{
+	if(up){
+		if(vexRT[Btn8R] == 1){
+			lowerCarrier();
+			up = false;
+		}
+	}
+	else{
```

Comment:
Clean this up to `else if { ... }` (unnecessary brackets and indentation)

Reply:
I'll do it after the competition